modules/svg/rapid_features.js

```diff
@@ -356,6 +379,46 @@ export function svgRapidFeatures(projection, context, dispatch) {
   }
 
 
+  function drawViewfieldPoints(selection, viewfieldPoints) {
+    let viewfield = selection.selectAll("g.suggestionViewfieldgroup")
+      .data(viewfieldPoints.length ? [0] : []);
+    viewfield.exit().remove();
+
+    viewfield = viewfield.enter()
+      .append('g')
+      .attr('class', 'suggestionViewfieldgroup')
+      .merge(viewfield);
+
+    let points = viewfield
+      .selectAll('g.viewfield')
+      .data(viewfieldPoints, d => d.key);
+
+
+    const enter = points.enter()
+      .append('g')
+      .attr('class', d => `viewfield ${d.key}`);
+
+
+    enter
```

Comment:
I guess the circle here needs to fit well to the shape of `viewfieldPath`? If so, let's add a comment here to emphasize that.